node/cn/api_tracer.go

```diff
@@ -55,6 +56,8 @@ const (
 	// and reexecute to produce missing historical state necessary to run a specific
 	// trace.
 	defaultTraceReexec = uint64(128)
+
+	fastCallTracer = "fastCallTracer"
```

Comment:
Please add some comment about this fast Call Tracer. :)

Reply:
Applied. 64c4e21